mobile/build.gradle

```diff
@@ -17,7 +17,7 @@ buildscript {
 }
 
 apply plugin: 'com.android.application'
-if(!isFoss) {
+if (!isFoss) {
```

Comment:
In a first glance they look correct anyway. One could put them to a separate commit, but since we squash PRs anyway...